<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber_Passed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Left">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>11</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>520</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начать тест</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_End">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>520</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Закончить</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>1231</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Вопросы</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Q1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Q2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Q3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Q4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Q5">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Question">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>1231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вопрос</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>931</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Ответы</string>
    </property>
    <widget class="QRadioButton" name="radioButton_A1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>921</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_A2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>921</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_A3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>921</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_A4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>921</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Answer">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>230</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ответить</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>240</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_SetName">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите имя:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Name">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>240</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ввести</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1275</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
